src/accelerate/big_modeling.py

```diff
@@ -387,8 +387,12 @@ def dispatch_model(
         retie_parameters(model, tied_params)
     else:
         device = list(device_map.values())[0]
-        if device != "disk":
+        # for backward compatibility
+        is_quantized = getattr(model, "is_quantized", False) or getattr(model, "is_loaded_in_8bit", False)
```

Comment:
`getattr(model, "is_loaded_in_8bit", False)` --> for backward compatilibty for users that have an old version of transformers (before 4bit integration)